### PR TITLE
docs: update README with maintainer names

### PR DESCRIPTION
**Pull request by lance**
## Proposed Changes

This commit changes the README to include all currently active
contributors to and the originator of the repository.

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

**Review by lholmquist — APPROVED** [review]

**Review by grant — APPROVED** [review]
- I'm not sure if Fabio has maintained this repo in the last year. Ping him?
- @lance your name should probably be first given you're most active and probably on Slack more

Otherwise looks good

**Comment by lance** [comment]
> @lance your name should probably be first given you're most active and probably on Slack more

I just put them in alphabetical order by first name. I'll reach out to Fabio

**Comment by lance** [comment]
@cloudevents/sdk-javascript-maintainers I have removed Fabio from the list of people to reach out to on Slack (tried pinging him on CNCF Slack with no response), and put maintainer names in alphabetical order by last name. Can I get a LGTM?

**Review by lholmquist — APPROVED** [review]

**Pull request merged**
